general/administration/hardware-acceleration.md

```diff
@@ -6,7 +6,31 @@ title: Hardware Acceleration
 
 # Hardware Acceleration
 
-Jellyfin supports hardware acceleration of video encoding/decoding/transcoding using FFMpeg. It supports multiple acceleration types, including  AMD AMF, Intel Quick Sync, OpenMax OMX, nVidia NVENC, Intel/AMD VAAPI, and others.
+Jellyfin supports hardware acceleration of video encoding/decoding using FFMpeg. FFMpeg can support multiple hardware acceleration implementations like Intel Quicksync (QSV), AMD AMF, OpenMax OMX, nVidia NVENC/NVDEC through the Video Acceleration API (VAAPI), and others.
+
+OS | Recommended HW Acceleration
+------------ | -------------
+Linux/GNU | VAAPI (recommended), NVENC, QSV, AMF
+Windows| QSV, NVENC, AMF, VAAPI
+MacOS| None (videotoolbox support coming)
+Android| MediaCodec, OMX
+RPi|OMX
```

Comment:
Could you use consistent spacing between the `|` characters in this table? One on each side works.